openstef/model/regressors/arima.py

```diff
@@ -0,0 +1,163 @@
+# SPDX-FileCopyrightText: 2017-2022 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+"""This module contains the SARIMAX regressor wrapper around statsmodels implementation."""
+import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+
+from sklearn.metrics import r2_score
+from sklearn.model_selection import TimeSeriesSplit
+from openstef.model.regressors.regressor import OpenstfRegressor
+
+
+class ARIMAOpenstfRegressor(OpenstfRegressor):
+    """Wrapper around statmodels implementation of (S)ARIMA(X) model.
+
+    To make prediction, it needs to update its historic, applying the fitted parameters to new data unrelated to the
```

Comment:
```suggestion
    To make a prediction, it needs to update its historic, applying the fitted parameters to new data unrelated to the
```

Reply:
I do not understand the word historic here, do you mean historic data?

Reply:
Yes I enhanced the documentation, it should be clearer.